x-pack/test/functional/services/machine_learning/job_wizard_common.ts

```diff
@@ -21,6 +26,16 @@ export function MachineLearningJobWizardCommonProvider({ getService }: FtrProvid
     return !subSelector ? subj : `${subj} > ${subSelector}`;
   }
 
+  function additionalSettingsSectionSelector(subSelector?: string) {
```

Comment:
This function is not needed here. The similar `advancedSectionSelector` was introduced because some element are hidden in the advanced section for some wizards, but are there right away in a different wizard.
Here you can use the `data-test-subj` in your service methods directly.

Reply:
Check https://github.com/elastic/kibana/pull/52269/commits/dc359ba43bfcafb6702441408bb6456e6d380741